modules/statics/src/coins.ts

```diff
@@ -868,21 +868,21 @@ export const coins = CoinMap.fromCoins([
     '5f9506c5-f10a-43c2-92d3-52941083bbc7',
     'injective',
```

Comment:
Not sure why this was changed. Our pattern would be INJ and inj. Same with Networks.main.inj

Reply:
I would recommend keeping to established patterns and fixing these.

Reply:
We have inj coin name already used for erc20 inj token. That is why we have used injective for this.

Reply:
Gotcha. Another name collision :(